release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -2717,3 +2717,19 @@ With this update, `aws-sdk-go` is bumped to v1.38.35, which supports new Amazon
 ==== Upgrading
 
 To upgrade an existing {product-title} 4.7 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#update-service-overview_updating-cluster-cli[Updating a cluster by using the CLI] for instructions.
+
+[id="ocp-4-7-22"]
+=== RHBA-2021:2903 - {product-title} 4.7.22 bug fix update
+
+Issued: 2021-08-03
+
+{product-title} release 4.7.22, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2021:2903[RHBA-2021:2903] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2021:2904[RHBA-2021:2904] advisory.
```

Comment:
There's no clause here, so you don't need the comma after the version. 

Also, it's just boilerplate being carried over, but I find this sentence hard to understand: "The list of bug fixes that are included in the update is documented..." _what_ is documented? The list, the bug fixes, the update? Could it just say, "The bug fixes included in the update are documented..."? (I'm a bit crotchety about the use of "that are" for clarity/localization in these cases, so by all means feel free to disregard that omission. 😁 )